pkgs/tools/cd-dvd/cdrtools/default.nix

```diff
@@ -20,6 +20,8 @@ stdenv.mkDerivation rec {
     ln -sv i386-darwin-clang64.rul RULES/arm64-darwin-cc.rul
     ln -sv i386-darwin-clang64.rul RULES/arm64-darwin-clang.rul
     ln -sv i386-darwin-clang64.rul RULES/arm64-darwin-clang64.rul
+  '' + lib.optionalString stdenv.hostPlatform.isRiscV64 ''
+    ln -sv i586-linux-cc.rul RULES/riscv64-linux-cc.rul
```

Comment:
Can we do these unconditionally? Same for those above.

Reply:
Yeah, should not be a problem.